### PR TITLE
README.md: Added License and Maintainers section

### PR DESCRIPTION
**Pull request by lohani2280**
fixes https://github.com/Murali-group/GraphSpace/issues/363

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,3 +102,18 @@ Contributing
...
+Maintainers
+=================
+
+The Project is maintained by :-
+
+- **T.M Murali ([@tmmurali](https://github.com/tmmurali))**
+- **Aditya Bharadwaj ([@adbharadwaj](https://github.com/adbharadwaj))**
+
+
+Liscence
```
Comment (tmmurali) [inline README.md]: Fix the spelling, please.

File: README.md [hunk 2]
```diff
@@ -102,3 +102,18 @@ Contributing
 =================
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Maintainers
+=================
+
+The Project is maintained by :-
```
Comment (tmmurali) [inline README.md]: Just say "If you have questions or suggestions about GraphSpace, please contact" Remove the ":-".

File: README.md [hunk 3]
```diff
@@ -102,3 +102,18 @@ Contributing
 =================
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Maintainers
```
Comment (tmmurali) [inline README.md]: Change to "Contact". Make sure it is a section header.

**Comment by lohani2280** [comment]
@tmmurali Thank You so much for the review. I have updated the PR as suggested. Please have a look at it.

File: README.md (hunk 1 above)
Comment (lohani2280) [reply README.md]: Really sorry for this. I have corrected.

File: README.md [hunk 4]
```diff
@@ -102,3 +102,18 @@ Contributing
...
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Contact
+=================
+
+If you have questions or suggestions about GraphSpace, please contact
+
+- **T.M Murali ([@tmmurali](https://github.com/tmmurali))**
```
Comment (tmmurali) [inline README.md]: Please change my name to "T. M. Murali" (with a period after the "M").

Comment (lohani2280) [reply README.md]: Sorry.I have made the correction.

**Comment by adbharadwaj** [comment]
👍

**Pull request merged**